### PR TITLE
[FIX] account_voucher_tax_sat: Added round of amount to 2 decimals in Pay Sat

### PR DESCRIPTION
**Pull request by luiseevaquer**
This PR is to fix https://github.com/Vauxoo/lodigroup/issues/681

### CODE REVIEW

File: account_voucher_tax_sat/model/account_voucher_tax_sat.py [hunk 1]
```diff
@@ -113,9 +113,9 @@ def action_close_tax(self):
         move_id = self.create_move_sat()
         self.write({'move_id': move_id.id})
 
-        amount_tax_sat = sum([
+        amount_tax_sat = round(sum([
             move_line_tax_sat.credit
-            for move_line_tax_sat in self.aml_ids])
+            for move_line_tax_sat in self.aml_ids]), 2)
```
Comment (JulioSerna) [inline account_voucher_tax_sat/model/account_voucher_tax_sat.py]: I suggest you to use the round of currency model something [like this ](https://github.com/odoo/odoo/blob/8.0/addons/account/account_invoice.py#L745)

**Comment by luiseevaquer** [comment]
@JulioSerna 

I made the change that you suggested, could you do technical review, please

**Review by JulioSerna — APPROVED** [review]

**Comment by luiseevaquer** [comment]
@JulioSerna 

Could you do technical review because I made other changes to make other rounding that I missed

**Comment by JulioSerna** [comment]
👍

**Comment by luiseevaquer** [comment]
@JulioSerna 

Could you merge this PR, please

**Comment by luiseevaquer** [comment]
@JulioSerna 

Here is the video showing in instance test how to round and balance all journal items After pressing "Liquidar Cuentas"

ask video to @lescobarvx "FIx issue 3472 - Error when reconciling Pago Sat"

**Review by JulioSerna — APPROVED** [review]

**Pull request merged**